docs/_documentations/che-installinfo.md

```diff
@@ -14,39 +14,101 @@ Install Che to use with Codewind or prepare to use Codewind with an existing Che
+4. Generate a Kubernetes secret containing the TLS secret and key you generated in the previous set:
+   ```
+   $ kubectl create secret tls che-tls --key=domain.key --cert=domain.crt -n che
+   ```
+5. Generate a Kubernetes secret containing the certificate you generated in step 2:
+   ```
+   $ cp rootCA.crt ca.crt
+   $ kubectl create secret generic self-signed-certificate --from-file=ca.crt -n che
+   ```
+6. Run the following command to install Che: 
+   ```
+   $ chectl server:start --platform=k8s --installer=operator --domain=<ingress-domain> --che-operator-cr-yaml=codewind-checluster.yaml --che-operator-image=quay.io/eclipse/che-operator:7.9.0
+   ```
 
 ## Updating an existing Che installation
 
-If you already have a Che installation, you can update it for Codewind.
+If you already have a Che installation with TLS, you can update it for Codewind.
 
 After creating the Codewind ClusterRole from the [Prerequisites](#prerequisites), run the following command, where `$NAMESPACE` is the namespace that your Che workspaces run in. By default, this namespace is `che`.
 ```
-kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.9.0/setup/install_che/codewind-rolebinding.yaml -n $NAMESPACE
+$ kubectl apply -f https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.10.0/setup/install_che/codewind-rolebinding.yaml -n $NAMESPACE
+```
+
+## Adding Certificates for Che to your browser
+
+If Eclipse Che was configured with self-signed certificates (such as by following the previous steps for Kubernetes, or if you installed on an OpenShift cluster with self-signed-certificates), you need to add the `ca.crt` for Eclipse Che to your browser. These steps are **not** necessary if you installed Che with publicly signed certificates (such as on OpenShift on IBM Cloud).
+
+### Download the OpenShift Router ca.crt
+If running on OpenShift with self-signed certs, follow these instructions:
+
+1. Authenticate against your OpenShift cluster, or ask your cluster administrator to do so.
+2. Run the following command to download the router's ca.crt
+```
+$ oc get secret router-ca -n openshift-ingress-operator -o jsonpath="{.data.tls\.crt}" | base64 -d > rootCa.crt
 ```
 
+### Adding the ca.crt to Google Chrome
+
+On MacOS, follow these steps:
+
+1. Open **Keychain Access** and click **File** -> **Import items**.
+2. Locate rootCa.crt you got earlier and import it.
+3. Find the certificate in Keychain and click on it.
+4. In the window that opens, expand the **Trust section** and under **When using this certificate**, select **Always trust**. Hit **save**.
+5. Reload Eclipse Che in your browser.
+
+On Windows, follow these steps:
+
+1. Open Google chrome preferences, select **Privacy and Security** and click on **Manage Certificates**.
+2. In the Window that pops up, click on the **Trusted Root Certificate Authorities** tab.
```

Comment:
- Lower case "window."
- "Up" will appear in our accessibility check. Better to have "appears" or something instead.
- "click" instead of "click on"